cherryontech/jupiter-jumpers
# Update mood log

## Description

1) Update the mood log modal:
- Added the icons, and modified the existing cards to add the subtitle for each weather
- Added a circular checkbox, and styling for when the card is selected
- Added the date for the current log
- Added a progress bar

## Related Issue

[SCRUM 18](https://cherryontech-jupiter-jumpers.atlassian.net/browse/SCRUM-18)

## Related Story Card


## Type of Changes

Styling updates

## Acceptance Criteria

- [X] Given I am on the log creation screen
- [X] When I select one of the five weather icons
- [X] Then the selected icon is highlighted and my mood is recorded

## Update Screenshots

_Provide screenshots of the view before and after the changes._

### Before
![Screenshot 2024-05-11 at 10 22 29 PM](https://github.com/cherryontech/jupiter-jumpers/assets/47403135/6be31f79-5075-4644-9c83-ff6c97c0aef9)

### After
![Screenshot 2024-05-11 at 10 19 32 PM](https://github.com/cherryontech/jupiter-jumpers/assets/47403135/853c15cf-0a78-4d1b-9511-907931800d09)

## Testing Instructions

(Open the console)
1. Initiate a new log
2. Select a weather icon
3. Ensure the weather icon's checkbox is checked, and a shadow and border appears on it
4. Attempt to select different weather icon cards to ensure the same thing as in #3 
5. Attempt closing the modal without registering the mood
6. Re-open the log, and attempt to select and press "continue"

NOTE: This styling update was done on a branch BEFORE Gina's bug fix, so we can't check that the bug fix still works here. We'll have to double check that after merge!


## Learnings (Optional)

_Document any things you learned or 'gotchas' you experienced._
